client/podman/informer.go

```diff
@@ -0,0 +1,148 @@
+package podman
+
+import (
+	"log"
+	"time"
+
+	"github.com/skupperproject/skupper/client/container"
+	"github.com/skupperproject/skupper/pkg/utils"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	resyncPeriod = time.Second * 5
+)
+
+//
+// Base types
+//
+
+type informerBase struct {
+	cli          *PodmanRestClient
+	resyncPeriod time.Duration
+}
+
+func (i *informerBase) SetResyncPeriod(t time.Duration) {
+	i.resyncPeriod = t
+}
+
+type updateNotification[T any] struct {
+	oldObj T
+	newObj T
+}
+
+type EventHandler[T any] interface {
```

Comment:
Just a detail regarding naming; the project has an interface called `EventHandlerInterface` that is implemented by components that generate events in skupper or in kubernetes with different methods to the `EventHandler` interface. What should we do in this case? using different names to express a difference in functionality (in this case methods)? I don't have a preference on which interface should be renamed.

Reply:
It makes sense.

I believe it might be clearer to move this new one under `client/container` and rename it to `Informer`,
this way it would be referred to by other packages (like podman or an eventual docker) as `container.Informer`. 

Check it out one more time, please.